mlir/lib/Conversion/SPIRVToLLVM/SPIRVToLLVM.cpp

```diff
@@ -33,7 +33,7 @@ using namespace mlir;
 // Constants
 //===----------------------------------------------------------------------===//
 
-static constexpr unsigned defaultAddressSpace = 0;
+constexpr unsigned defaultAddressSpace = 0;
```

Comment:
Why removing "static" ? This global variable is used within this file only.

Reply:
`constexpr` already implies static linkage

Reply:
It was nitpicking, also, other `static constexpr` can be found all around the file

Reply:
ok, somehow I forgot about that.  I always prefer to use "static" in situation like this one just to make it explicit.